tests/python_on_whales/components/test_service.py

```diff
@@ -45,6 +45,25 @@ def test_get_list_of_services(docker_client: DockerClient):
         assert [my_service] == list_of_services
 
 
+@pytest.mark.usefixtures("swarm_mode")
+def test_filters(docker_client: DockerClient):

```

Comment:
Can you try to filter with a label that does not exist and verify that no services are returned? That would ensure that the function is correctly removing services from the result.